crates/punktf-cli/main.rs

```diff
@@ -251,6 +251,12 @@ fn handle_commands(opts: opt::Opts) -> Result<()> {
 					if deployment.status().is_failed() {
 						Err(eyre!("Some dotfiles failed to deploy"))
 					} else {
+						if options.dry_run {
```

Comment:
I would put this check before the failed check on line 251. That way this info is also printed if an error occured.

Reply:
True.